xtask/src/flash_image.rs

```diff
@@ -0,0 +1,504 @@
+// Licensed under the Apache-2.0 license
+
+use crate::DynError;
+use crc32fast::Hasher;
+use std::fs::File;
+use std::io::{self, Read, Write};
+use std::io::{Error, ErrorKind};
```

Comment:
I think you can can combine these imports.

Reply:
yes, will do